dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-http/src/main/java/org/apache/dolphinscheduler/plugin/alert/http/HttpAlertChannelFactory.java

```diff
@@ -79,7 +79,15 @@ public List<PluginParams> params() {
                                 .build())
                         .build();
 
-        return Arrays.asList(url, requestType, headerParams, bodyParams, contentField);
+        InputParam timeout =
```

Comment:
Please use InputNumberParam` is better, and use `setValue1 to set default value to 120.

Reply:
Done